### DISCUSSION TITLE
From Beta to free community version - project impact

### DISCUSSION THREAD

**Discussion by bonchevapavlina** [opengisch/QField#3900] [Q&A]
Hello,
Congratulations on ending the Beta status!
In the last year I have been using QFieldCloud for storing and processing a single project - https://app.qfield.cloud/a/ViKEOOD-Sofia/ViKSofia/
After reviewing most of the announcements about the upcoming transition I am still not clearly aware of how exactly the changes will affect the way we are working with that specific project. And also what are the possibilities in front, in terms of keeping our workflow unchanged.
Currently we use the project to gather onsite data. The project covers multiple so called areas/regions. The data in each separate area is added by a dedicated person on site who is adding located objects via the mobile app. The project is private and has 26 contributors at the moment.

Ending the Beta, opens a couple of questions for me:
- Will the project be converted from private to public automatically?
- Is there any difference if the project is owned by organization or by a user? 
- Can we keep the workflow as is if we proceed with the community version? If not, which package will fit best our current needs and how much would it cost?

We love the product but we I am not sure if we will be able to afford it and how viable it will be for us to continue using it.

**Comment by suricactus** [comment]
Hello, @bonchevapavlina . Than you for being with QFieldCloud during the beta phase!

Let me try to answer your questions.

> Will the project be converted from private to public automatically?

No, projects remain as they are. If you havent upgraded by 22nd February, you can still download your project files, but you cannot interact with QFieldSync/QField. If you upgrade, you instantly get the functionality as usual. If you don't, you either convert your projects to public or you have the option to delete your project files until you fit in the community plan.

> Is there any difference if the project is owned by organization or by a user?

The main difference is that organization private projects are easy to be shared across multiple people, while user owned private projects can be shared with maximum 1 other person.

> Can we keep the workflow as is if we proceed with the community version? If not, which package will fit best our current needs and how much would it cost?

Unless you convert your projects to public projects, you need to upgrade to paid plan. You can easily calculate the cost by multiplying the organization per user price with your monthly active users. Note there is minimum of 2 active users per month.

Check my explanations in my other post in the forum https://github.com/opengisch/QField/discussions/3836#discussioncomment-4772755 .


If you still need any clarifications, please ping me on ivan@opengis.ch. Говорим и на български ;) .

**Reply by bonchevapavlina** [reply]
Много благодаря за отговора!
И ние говорим не-лошо български ;)
